### DISCUSSION TITLE
Is github social media now?

### DISCUSSION THREAD

**Discussion by trobert2** [community/community#37334] [Other Feature Feedback, Questions, & Ideas]
This whole main page experience is bad. I don't come to github to see what some guy I opened a PR for 5 years ago is doing. Why do I care he created a new repo? Why do I get `sponsor` suggestions? I don't want to see that.

I come to github to get work done. let me do that or I'm switching to gitlab

**Comment by ossanna16** [comment]
Hi there @trobert2 and thank you for the feedback!

**Comment by Property404** [comment]
>some guy I opened a PR for 5 years ago is doing  

+1, it's so weird. If I wanted to follow them, I'd follow them

**Comment by vlachenal** [comment]
I am totally agreed with you ... GitHub home page is horrible !
I don't want to see lots of useless "cards" and recommendations to follow people I don't care about.
Please make GitHub minimalist again. Its original goal is not to be a social network !

**Comment by MasterInQuestion** [comment]
Online repository, and nothing else.
    .
    The rest, does not matter.

**Comment by rdbende** [comment]
Sadly it looks like it is. It's crazy how Microsoft can ruin its products.
